README.md

```
@@ -14,7 +15,12 @@ Handles user management at the platform level. It manages users and their relati
 
 ### WebApi
 The main entry point for the platform. It is a Minimal API project that hosts the modular monolith. It aggregates APIs from all domains and provides a unified interface.
-- **Documentation**: Uses Scalar (available at `/api-doc` in Development).
+- **Documentation**: Uses [Scalar](https://github.com/scalar/scalar) (available at `/api-doc`) for verifying the OpenAPI V3 specification. 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)